man/systemd-sleep.conf.xml

```diff
@@ -227,6 +227,23 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>HibernateOnACPower=</varname></term>
+
+        <listitem>
+          <para>Whether to allow hibernation when the system has AC power. Only used by
+          <citerefentry><refentrytitle>systemd-suspend-then-hibernate.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+          when <varname>HibernateDelaySec=</varname> is set.</para>
+
+          <para>If this option is disabled, once the system no longer has AC power (after <varname>SuspendEstimationSec=</varname>), the system
+          will delay <varname>HibernateDelaySec=</varname> before hibernating, or hibernate when battery is low.</para>
```

Comment:
The description isn't necessarily clear. I think some comparisons with the normal operation mode can be highlighted:

If ... disabled, the countdown of `HibernateDelaySec=` starts only after the AC power is disconnected, keeping the system in suspend state otherwise.

You get the idea ;-)